������middle project 4 - להגשה.py

```diff
@@ -0,0 +1,163 @@
+import pandas as pd
+import json
+import os
+from person_file import Person
+from student_class import Student
+from employee_class import Employee
+
+def errorNumber(name_value, value):
+    print("Error: " + name_value + " must be a number. " + value + " is not a number")
+
+def printDetails(id, name, age, space=""):
+    print(space + "ID: " + id)
+    print(space + "Name: " + name)
+    print(space + "Age: " + age)
+
+def saveNewEntry(people_dict):
+    id_input = input("ID: ")
+    if id_input.isdigit(): #check if id is a integer
+        if id_input in people_dict:
+            print("Error: ID already exists: " + str(people_dict[id_input]))
+            return 0
+    else:
+        errorNumber("ID", id_input)
+        return 0
+    
+    name_input = input("Name: ")
+    age_input = input("Age: ")
+    if age_input.isdigit() == False:
+        errorNumber("Age", age_input)
+        return 0
+    
+    type_input = input(name_input + " is a person/ student/ employee? ")
+    if type_input == "student":
+        field_of_study = input("Field of study: ")

```

Comment:
תנסי לחשוב איך את מכניסה את האינפוטים של כל קלאס לתוך הקלאס עצמו.
אינפוט זה השלב שאתה מבקש מהמשתמש נתונים.
אני רוצה שהפונקציה saveNewEntry תהיה קצרה ולא יהיה בה למשל Year of study , אלא שזה יהיה בתוך הקלאס של הסטודנט.

הפרויקט הזה יכול לקחת עוד קצת זמן, זה בסדר.